tests/performance/specs/home.test.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * WordPress dependencies
+ */
+import { test } from '@wordpress/e2e-test-utils-playwright';
+
+/**
+ * Internal dependencies
+ */
+import { camelCaseDashes } from '../utils';
+
+const results = {
+	timeToFirstByte: [],
+	largestContentfulPaint: [],
+	lcpMinusTtfb: [],
+};
+
+const themes = [ 'twentytwentyone', 'twentytwentythree', 'twentytwentyfour' ];
+
+const locales = [ 'en_US', 'de_DE' ];
+
+test.describe( 'Front End', () => {
```

Comment:
I really like consolidating all of the use cases this way. Do you think we could extend this in the future to test different URLs (e.g., a single post) or would you suggest making a separate spec for any other use cases beyond the home page?

Reply:
Sure that could all be kept in the same file. I did something similar for https://github.com/swissspidy/wp-i18n-benchmarks